src/enkf/controlvec.f90

```diff
@@ -229,7 +229,7 @@ subroutine read_control()
    q_ind = getindex(cvars3d, 'q')
    if (nproc == 0) then
      t2 = mpi_wtime()
-     print *,'time in readgridata on root',t2-t1,'secs'
+     print *,'time in readgridata/readgrid_pnc on root',t2-t1,'secs'
```

Comment:
Personal preference:  timers should be placed around the section of code being timed.  This print provides timing for one of two possible subroutine calls.  

While it is more coding, I'd add a `nproc==0` block for `t2` with a print after each subroutine being timed.   

This is just a comment, not a request to change the code.